firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/UpdateAppClient.java

```diff
@@ -79,10 +109,185 @@ private void redirectToPlayForAabUpdate(String downloadUrl, Activity currentActi
     this.updateTask.updateProgress(updateState);
   }
 
+  public void updateApk(String downloadUrl, Activity currentActivity) {
+    getDownloadTask(downloadUrl)
+        .addOnSuccessListener(
+            file -> {
+              install(file.getPath(), currentActivity)
+                  .addOnSuccessListener(
```

Comment:
this success listener executes on the main thread by default, unless we pass in an executor. Is this what we want?